src/features/CitiesList/CityListItem.tsx

```diff
@@ -0,0 +1,57 @@
+import { memo } from 'react';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import Row from 'components/Row/Row';
+import WeatherIcon from 'components/WeatherIcon/WeatherIcon';
+import TemperaturePill from 'components/TemperaturePill/TemperaturePill';
+import Chevron from 'components/Chevron/Chevron';
+
+interface Props {
+  cityName: string;
+  conditions: string;
+  temp: number;
+  icon: string;
+  id: number;
+  onPress(id: number): void;
+}
+
+const CityListItem: React.FC<Props> = ({
+  cityName,
+  conditions,
+  temp,
+  icon,
+  id,
+  onPress,
+}) => {
+  const handleOnPress = () => onPress?.(id);
```

Comment:
Types suggest that onPress is always defined.
```suggestion
  const handleOnPress = () => onPress(id);
```

Reply:
just a typo